### PR TITLE
Switch type order of SearchReadsRequest.start

### PR DESCRIPTION
**Pull request by dcolligan**
See discussion that originated this PR in https://github.com/ga4gh/server/issues/362

### CODE REVIEW

File: src/main/resources/avro/readmethods.avdl [hunk 1]
```diff
@@ -30,7 +30,7 @@ record SearchReadsRequest {
   Requests spanning the join of circular genomes are represented as
   two requests one on each side of the join (position 0).
   */
-  union { long, null } start = 0;
+  union { null, long } start = 0;
```
Comment (fnothaft) [inline src/main/resources/avro/readmethods.avdl]: You need to change `0` to `null` for the default value to be correct here.

**Comment by dcolligan** [comment]
@fnothaft fixed

**Comment by jeromekelleher** [comment]
+1

**Comment by macieksmuga** [comment]
+1

**Comment by diekhans** [comment]
+1

this needs to be merged ASAP, as the schema is broken.

**Pull request merged**